integration-tests/jail/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN sed -i 's/security.ubuntu.com/ftp.daum.net/g' /etc/apt/sources.list
 RUN sed -i 's/archive.ubuntu.com/ftp.daum.net/g' /etc/apt/sources.list
 RUN apt-get update
 RUN apt install -y --no-install-recommends whois wget curl python3 libssl-dev libcurl4-openssl-dev libelf-dev libdw-dev  gcc binutils-dev libc6-dev pkg-config build-essential openssh-server cmake libgflags-dev libzmq3-dev pkg-config libssl-dev libzmq3-dev  unzip  tmux 
-ENV SGX_MODE=SW
+ENV SGX_MODE=HW
 ENV NETWORK_ID=AB
 ENV RUSTFLAGS=-Ctarget-feature=+aes,+ssse3
 ENV PATH=$HOME/bin:$PATH
```

Comment:
are these docker files needed?

Reply:
this is multiple nodes test, docker-compose is necessary.

Reply:
but one can just use one image (e.g. `cryptocom/chain:latest`) instead of building all these custom images and set this up in docker-compose with `command` and/or `entrypoint`:
https://docs.docker.com/compose/compose-file/#command
https://docs.docker.com/compose/compose-file/#entrypoint